### DISCUSSION TITLE
My first PR and Discussions

### DISCUSSION THREAD

**Discussion by iuryt** [euroargodev/argopy#192] [General]
Hi,

I would love to contribute to this amazing project.
The list of good first issues is empty. Any suggestions for me to help you out?

Also, have you considered opening the discussions tab for this repository?

**Comment by gmaze** [comment]
Hi @iuryt 

🤝  Thanks for your help !

Indeed, no issues have yet this label, I could give it a try to see where you could start...

In the mean time, our dev roadmap is here: https://github.com/euroargodev/argopy/milestone/3
As you can see, this is rather technical with regard to the Argo data stream, so I guess your contribution will depend on how familiar you are with Argo ?

> Also, have you considered opening the discussions tab for this repository?

We've thought about this but did not moved on yet, so let's use this PR as a ✨  !

**Comment by gmaze** [comment]
Hi @iuryt and other possible contributors:

I have labelled issues with labels like:

- **Good first issues**: https://github.com/euroargodev/argopy/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

- **Help wanted**: https://github.com/euroargodev/argopy/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

this should help to check where to start

I'm also very happy to answer any questions related to how to contribute to argopy !

In 2025, we will organise the first argopy dedicared workshop, it will be both a training camp for end users and a dev workshop for contributors, stay tuned.
